core/tauri/src/protocol/asset.rs

```diff
@@ -36,24 +37,18 @@ fn get_response(
     .decode_utf8_lossy()
     .to_string();
 
+  let mut resp = Response::builder().header("Access-Control-Allow-Origin", window_origin);
+
   if let Err(e) = SafePathBuf::new(path.clone().into()) {
     debug_eprintln!("asset protocol path \"{}\" is not valid: {}", path, e);
-    return Response::builder()
-      .status(403)
-      .body(Vec::new().into())
-      .map_err(Into::into);
+    return resp.status(403).body(Vec::new().into()).map_err(Into::into);
   }
 
   if !scope.is_allowed(&path) {
     debug_eprintln!("asset protocol not configured to allow the path: {}", path);
-    return Response::builder()
-      .status(403)
-      .body(Vec::new().into())
-      .map_err(Into::into);
+    return resp.body(Vec::new().into()).map_err(Into::into);
```

Comment:
is the missing 403 here (that was there before) intentional?

Reply:
that's just me being stupid, can't even copypasta stuff
thanks